<ui version="4.0">
 <author/>
 <comment/>
 <exportmacro/>
 <class>SendMessageToPagerSnppWidget</class>
 <widget class="QWidget" name="SendMessageToPagerSnppWidget">
  <property name="geometry">
  <rect>
    <x>0</x>
    <y>0</y>
    <width>455</width>
    <height>145</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <family>DejaVu Sans</family>
    <pointsize>8</pointsize>
   </font>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <property name="spacing">
    <number>6</number>
   </property>
   <property name="leftMargin">
    <number>15</number>
   </property>
   <property name="topMargin">
    <number>2</number>
   </property>
   <property name="rightMargin">
    <number>25</number>
   </property>
   <property name="bottomMargin">
    <number>5</number>
   </property>
   <item>
    <layout class="QFormLayout" name="formLayout">
     <property name="horizontalSpacing">
      <number>6</number>
     </property>
     <property name="verticalSpacing">
      <number>2</number>
     </property>
     <item row="0" column="0">
      <widget class="QLabel" name="lblPagerId">
       <property name="text">
        <string>Pager ID</string>
       </property>
       <property name="buddy">
        <cstring>cmbPagerId</cstring>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QComboBox" name="cmbPagerId">
       <property name="editable">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="lblMailTemplate">
       <property name="text">
        <string>Mail template</string>
       </property>
       <property name="buddy">
        <cstring>cmbMailTemplate</cstring>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <layout class="QHBoxLayout" name="horizontalLayout_2">
       <property name="spacing">
        <number>0</number>
       </property>
       <item>
        <widget class="QComboBox" name="cmbMailTemplate"/>
       </item>
       <item>
        <widget class="QToolButton" name="btnMailTemplate">
         <property name="text">
          <string>...</string>
         </property>
         <property name="icon">
          <iconset resource="../../qHostMonitor.qrc">
           <normaloff>:/img/foldersTree.png</normaloff>:/img/foldersTree.png</iconset>
         </property>
         <property name="iconSize">
          <size>
           <width>16</width>
           <height>16</height>
          </size>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item row="2" column="0">
      <widget class="QLabel" name="lblLevel">
       <property name="text">
        <string>Level</string>
       </property>
       <property name="buddy">
        <cstring>cmbLevel</cstring>
       </property>
      </widget>
     </item>
     <item row="2" column="1">
      <layout class="QHBoxLayout" name="horizontalLayout" stretch="4,1">
       <property name="spacing">
        <number>20</number>
       </property>
       <item>
        <widget class="QComboBox" name="cmbLevel">
         <item>
          <property name="text">
           <string>Priority</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Normal</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Five minutes</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Fifteen minutes</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>One hour</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Four hours</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Twelve hours</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Twenty Four hours</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Carrier specific &quot;1&quot;</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Carrier specific &quot;2&quot;</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Carrier specific &quot;3&quot;</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Carrier specific &quot;4&quot;</string>
          </property>
         </item>
        </widget>
       </item>
       <item>
        <widget class="QCheckBox" name="chkAlert">
         <property name="text">
          <string>Alert</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
   <item>
    <widget class="Line" name="line">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_3" stretch="6,4">
     <item>
      <layout class="QFormLayout" name="formLayout_3">
       <item row="0" column="0">
        <widget class="QLabel" name="lblServer">
         <property name="text">
          <string>Server</string>
         </property>
         <property name="buddy">
          <cstring>cmbServer</cstring>
         </property>
        </widget>
       </item>
       <item row="0" column="1">
        <widget class="QComboBox" name="cmbServer">
         <property name="editable">
          <bool>true</bool>
         </property>
         <item>
          <property name="text">
           <string>localhost</string>
          </property>
         </item>
        </widget>
       </item>
       <item row="1" column="0">
        <widget class="QCheckBox" name="chkLoginAs">
         <property name="text">
          <string>Login as</string>
         </property>
        </widget>
       </item>
       <item row="1" column="1">
        <widget class="QLineEdit" name="lineLoginAs"/>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QFormLayout" name="formLayout_2">
       <property name="labelAlignment">
        <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
       </property>
       <item row="0" column="1">
        <widget class="QSpinBox" name="spinPort">
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
         <property name="minimum">
          <number>1</number>
         </property>
         <property name="maximum">
          <number>65535</number>
         </property>
         <property name="value">
          <number>444</number>
         </property>
        </widget>
       </item>
       <item row="1" column="0">
        <widget class="QLabel" name="lblPassword">
         <property name="text">
          <string>password</string>
         </property>
         <property name="buddy">
          <cstring>linePassword</cstring>
         </property>
        </widget>
       </item>
       <item row="1" column="1">
        <widget class="QLineEdit" name="linePassword"/>
       </item>
       <item row="0" column="0">
        <widget class="QLabel" name="lblPort">
         <property name="text">
          <string>Port</string>
         </property>
         <property name="buddy">
          <cstring>spinPort</cstring>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="../../qHostMonitor.qrc"/>
  <include location="../../qHostMonitor.qrc"/>
 </resources>
 <connections/>
</ui>


